src/dlgConnectionProfiles.cpp

```diff
@@ -2299,40 +2299,30 @@ void dlgConnectionProfiles::loadProfile(bool alsoConnect)
         mudlet::self()->mDiscord.setApplicationID(pHost, mDiscordApplicationId);
     }
 
-    if (needsGenericPackagesInstall) {
-        // Install appropriate mapper and other scripts for the game
-        if (pHost->getUrl().contains(QStringLiteral("aetolia.com"), Qt::CaseInsensitive)
-            || pHost->getUrl().contains(QStringLiteral("achaea.com"), Qt::CaseInsensitive)
-            || pHost->getUrl().contains(QStringLiteral("lusternia.com"), Qt::CaseInsensitive)
-            || pHost->getUrl().contains(QStringLiteral("imperian.com"), Qt::CaseInsensitive)) {
-            // IRE MUD profiles:
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/mudlet-mapper.xml"));
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/send-text-to-all-games.xml"));
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/deleteOldProfiles.xml"));
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/echo.xml"));
-
-        } else if (pHost->getUrl().contains(QStringLiteral("carrionfields.net"), Qt::CaseInsensitive)) {
-            // Carrion Fields loader - downloads and installs latest GUI from GitHub site
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/CF-loader.xml"));
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/send-text-to-all-games.xml"));
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/deleteOldProfiles.xml"));
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/echo.xml"));
-
-        } else if (pHost->getUrl().contains(QStringLiteral("mudlet.org"), Qt::CaseInsensitive)) {
-            // Mudlet self-test profile ONLY:
-            mudlet::self()->packagesToInstallList.append(QStringLiteral(":/run-tests.xml"));
+    if (preIntallPackages) {
+        auto gameUrl = pHost->getUrl().toLower();
+        const QHash<QString, QStringList> defaultScripts = {
```

Comment:
Using a hash table - doesn't that mean that the order of the packages is not well specified - so that the order in which they are loaded into the profile is not predictable - will that be an issue if, say one of the mapper packages is inserted before the `send-text-to-all-games` one or the `deleteOldProfiles` is loaded after all the others and one of them make use of it in their start-up.  I know these are perhaps artificially contrived scenarios, but is the load order significant or not?

Reply:
The load order is not significant.